CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Next
```

Comment:
This should be an actual version number, and `VERSION` should be updated.

Reply:
Okay, bumping revision by 1